docs/proposals/job_controller.md

```diff
@@ -0,0 +1,192 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pods that require to run:
+* once on demand or at a specified point in time
```

Comment:
Why do these have to be part of the same object design?  A scheduled job creates multiple jobs at different times.  A single job has no link to other jobs (a scheduled job may want to be exclusive if a job is already running, but jobs can provide no such guarantee).

Want to see a justification for this being a single object vs Job and ScheduledJob